scheduler/README.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Scheduler Pattern
+category: Creational
+language: en
+tag:
```

Comment:
Please pick suitable tags from https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags